onecodex/viz/_pca.py

```diff
@@ -154,6 +163,7 @@ def plot_pca(
             ylabel = "PC2 ({}%)".format(round(pca.explained_variance_ratio_[1] * 100, 2))
 
         # don't send all the data to vega, just what we're plotting
+        # we need to filter down to just the correct IDs
         plot_data = pd.concat(
             [pca_vals.loc[:, ("PC1", "PC2")], magic_metadata], axis=1
         ).reset_index()
```

Comment:
Can't comment on the exact line: can `onecodex/viz/base_viz_mixin.py` be removed? It's empty